source/features/add-commented-menu-item.js

```diff
@@ -0,0 +1,25 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from '../libs/page-detect';
+import {getUsername} from '../libs/utils';
+
+export default function () {
+	const pageName = pageDetect.isIssueSearch() ? 'issues' : 'pulls';
+	const type = pageDetect.isIssueSearch() ? 'issue' : 'pr';
+	const username = getUsername();
+
+	if (select.exists('.refined-github-commented')) {
```

Comment:
This should be at the top of the function, so we do the least amount of work if it already exists.